### ISSUE TITLE
NoMethodError: undefined method `version' for nil:NilClass

### ISSUE THREAD

**Issue by desoleary** [palkan/test-prof#49]
### Tell us about your environment

**Ruby Version:**
> ruby 2.2.6p396

**Framework Version (RSpec, Minitest, FactoryGirl, Rails, whatever):**
> RSpec (via rspec-rails v 3.6.0)

**TestProf Version:**
> test-prof (0.4.4)

### What did you do?
> ran specs when using `let_it_be`

### What did you expect to happen?
> not to fail :)

### What actually happened?
```
NoMethodError:
  undefined method `version' for nil:NilClass
```
This is due to the fact that it is looking for 'rspec' gem specifically instead of 'rspec' or 'rspec-rails'

**Comment by guilpejon** [comment]
+1 Really needed this fix today

**Comment by palkan** [comment]
Fixed and released as 0.4.6

**Comment by guilpejon** [comment]
Thank you @palkan!
